### PR TITLE
raidboss: improve sigmascape normal triggers

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: ui/raidboss/data/04-sb/raid/o6n.ts [hunk 1]
```diff
@@ -24,19 +37,36 @@ const triggerSet: TriggerSet<Data> = {
...
-      id: 'O6N Meteors',
+      id: 'O6N Demonic Pain',
+      type: 'Tether',
+      // 0001 = far enough, 0039 = too close
+      netRegex: NetRegexes.tether({ id: ['0001', '0039'] }),
+      condition: (data, matches) => data.me === matches.target || data.me === matches.source,
+      suppressSeconds: 10,
+      alertText: (data, matches, output) => {
+        const partner = data.me === matches.target ? matches.source : matches.target;
+        return output.text!({ player: data.ShortName(partner) });
```
Comment (JLGarber) [inline ui/raidboss/data/04-sb/raid/o6n.ts:48]: Isn't this always a tether between the player and the boss on normal mode?

File: ui/raidboss/data/04-sb/raid/o7n.ts [hunk 2]
```diff
@@ -23,12 +54,12 @@ const triggerSet: TriggerSet<Data> = {
...
-          fr: 'Rayon Magitek',
-          ja: '魔導レーザー',
-          cn: '直线AOE',
-          ko: '마도 레이저',
+          en: 'Get out of front',
+          de: 'Weg von vorn',
+          fr: 'Partez du devant',
+          ja: '正面から離れる',
+          cn: '远离正面',
+          ko: '정면 피하기',
```
Comment (JLGarber) [inline ui/raidboss/data/04-sb/raid/o7n.ts:62]: Huhhhh, we *don't* have a canned response for this. I'm honestly a little surprised!

File: ui/raidboss/data/04-sb/raid/o7n.ts [hunk 3]
```diff
@@ -71,6 +100,55 @@ const triggerSet: TriggerSet<Data> = {
...
+      netRegex: NetRegexes.startsUsing({ id: '2752', source: 'Guardian', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '2752', source: 'Wächter', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '2752', source: 'Gardien', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '2752', source: 'ガーディアン', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '2752', source: '守护者', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '2752', source: '가디언', capture: false }),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Activate 3 person towers',
```
Comment (JLGarber) [inline ui/raidboss/data/04-sb/raid/o7n.ts:131]: This is probably as short as we can make any shot call for this one. Maybe "get 3-person towers", but it's all the same.

**Review by JLGarber — APPROVED** [review]

File: ui/raidboss/data/04-sb/raid/o7n.ts (hunk 3 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/raid/o7n.ts:121]: Yeah, I was not that happy with this either.  This was the least worst idea I could come up with.

File: ui/raidboss/data/04-sb/raid/o7n.ts (hunk 2 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/raid/o7n.ts]: Oh hmm, we do have `awayFromFront`.  I'll use that.

File: ui/raidboss/data/04-sb/raid/o6n.ts (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/raid/o6n.ts]: Thank you for catching this!!

**Pull request merged**